dash_uploader/upload.py

```diff
@@ -143,6 +143,6 @@ def Upload(
     )
 
     if filetypes:
-        arguments['filetypes'] = filetypes
+        arguments['fileTypes'] = filetypes

```

Comment:
Is there a bug here? I think the `Upload_ReactComponent` works with the lower case "t".